wwwroot/data/regionMapping.json

```diff
@@ -307,13 +307,36 @@
             "regionProp": "ELECT_DIV",
             "textCodes": true,
             "aliases": [
-                "divisionnm",
-                "ced_aec",
                 "ced_aec_2011"
             ],
             "description": "Commonwealth electoral districts (Australian Electoral Commission)",
             "regionIdsFile": "data/regionids/region_map-FID_COM20111216_ELB_region_ELECT_DIV.json"
         },
+        "CED_AEC_2016": {
+            "layerName": "region_map:FID_COM20160509_ELB",
+            "server": "http://geoserver.nationalmap.nicta.com.au/region_map/ows",
+            "regionProp": "DIV_ID",
```

Comment:
I think we need to consider the broader application of electoral boundaries (including historical ones,state/territory lower house and  state upper houses), and therefore replace the aec with "com". I'd also like to avoid CED where possible as this is a very-ABS specific term, referring to the ABS approximated boundaries (likewise SED).

Possibly:
elect_div | com_elb_name_2016 | com_elb16 | divisionnm
div_id | com_elb_code_2016 | 

(leading to nsw_elb_code_2016 and potentially the nsw_elb_u_code_2016 for upper house boundaries where applicable - more to come from Admin Boundaries soon)

Happy to talk it through?